### PR TITLE
DSL for managing New Relic server alerts.

### PR DESCRIPTION
**Pull request by feanil**

### CODE REVIEW

File: devops/resources/manage-newrelic-server-alerts.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+cd sysadmin/newrelic
+
+pip install -r requirements.txt
+
+python add_app_servers_to_server_policy.py --new-relic-api-key ${NEW_RELIC_API_KEY} --alert-policy-name ${ALERT_POLICY_NAME} ${APP_NAMES}
```
Comment (jibsheet) [inline devops/resources/manage-newrelic-server-alerts.sh]: Hannah and I fixed this at the system level, but just confirm that the api key gets *****ed in the jenkins console

File: devops/jobs/ManageNewRelicServerAlerts.groovy [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+                logRotator common_logrotator
+                wrappers common_wrappers
+
+                wrappers {
+                    credentialsBinding {
+                        string('NEW_RELIC_API_KEY', 'new-relic-api-key')
+                    }
+                }
+
+                def gitCredentialId = extraVars.get('SECURE_GIT_CREDENTIAL_ID')
```
Comment (jibsheet) [inline devops/jobs/ManageNewRelicServerAlerts.groovy]: You make this var, but down below

File: devops/jobs/ManageNewRelicServerAlerts.groovy [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+                    stringParam('SYSADMIN_BRANCH', extraVars.get('SYSADMIN_BRANCH', 'master'),
+                            'e.g. tagname or origin/branchname')
+                }
+
+                multiscm {
+                    git {
+                        remote {
+                            url('$SYSADMIN_REPO')
+                            branch('$SYSADMIN_BRANCH')
+                            credentials(extraVars.get("SECURE_GIT_CREDENTIAL_ID"))
```
Comment (jibsheet) [inline devops/jobs/ManageNewRelicServerAlerts.groovy]: this should use gitCredentialId or delete that var

File: devops/resources/manage-newrelic-server-alerts.sh (hunk 1 above)
Comment (feanil) [inline devops/resources/manage-newrelic-server-alerts.sh]: It did in my testing, yes.

**Pull request merged**